package.json

```diff
@@ -3,6 +3,14 @@
   "version": "1.1.13",
   "description": "Streams3, a user-land copy of the stream library from Node.js v0.11.x",
   "main": "readable.js",
+  "files": [
```

Comment:
we fundamentally disagree. node_modules is part of my application.

It's incredibly important for all the information to be there.

If your using npm and node_modules to publish "binary libraries" and complain about the size, maybe have less modules.